RecoMET/METAlgorithms/src/EcalHaloAlgo.cc

```diff
@@ -60,14 +60,11 @@ EcalHaloData EcalHaloAlgo::Calculate(const CaloGeometry& TheCaloGeometry, edm::H
       
       // Get Det Id of the rechit
       DetId id = DetId(hit->id()); 
-      const CaloSubdetectorGeometry* TheSubGeometry = nullptr;                                                                         
-      const CaloCellGeometry* cell = nullptr ;                                                                                    
 
       // Get EB geometry 
-      TheSubGeometry = TheCaloGeometry.getSubdetectorGeometry(DetId::Ecal, 1);                                                         
+      const CaloSubdetectorGeometry* TheSubGeometry = (TheCaloGeometry.getSubdetectorGeometry(DetId::Ecal, 1));                                                         
```

Comment:
@bsunanda - please, change the name to lower case and there is no need for extra brackets